homeassistant/components/logbook.py

```diff
@@ -239,6 +259,38 @@ def humanify(events):
                     entity_id)
 
 
+def _exclude_events(events, config):
+    """Get lists of excluded entities and platforms."""
+    excluded_entities = []
+    excluded_domains = []
+    exclude = config[DOMAIN].get(CONF_EXCLUDE)
+    if exclude:
+        excluded_entities = exclude.get(CONF_ENTITIES, [])
```

Comment:
I see now that this check is needed because your tests contain the wrong config. Best to run your `EMPTY_CONFIG` through `CONFIG_SCHEMA` so defaults get set (and it keeps working in the future).